### DISCUSSION TITLE
Sponsoring

### DISCUSSION THREAD

**Discussion by syroy** [cristianbuse/VBA-FileTools#27] [General]
Hi @cristianbuse . I would like to sponsor you for the VBA-FileTools code, but whenever I enter an amount the "Select" button greys out and I cannot continue. Would you know why? I have been able to sponsor other repos so really wondering...

**Comment by cristianbuse** [comment]
Hi @syroy ,

Unfortunately, I do not know how it's supposed to work. I just tried it myself and it does not grey out. Maybe try toggling between Montly and One-Time and see if that fixes it. If it doesn't work, don't worry! Thank you anyway!
